client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -41,7 +43,7 @@ const (
 	// dockerUSocket: Unix socket on which Docker engine is listening
 	dockerUSocket = "unix:///var/run/docker.sock"
 	// Postfix added to names of Samba services for volumes
-	serviceNamePostfix = "fileServer"
```

Comment:
This seems works for me. I can remove the underscore. But do you know which component the DNS complaint is coming from?